builder/rust/cargo/project_aspect.bzl

```diff
@@ -211,24 +211,47 @@ def _get_properties(target, ctx, source_files, crate_info):
         properties["edition"] = ctx.rule.attr.edition or _DEFAULT_RUST_EDITION
         entry_point_file = _entry_point_file(target, ctx, source_files)
         properties["entry.point.path"] = _src_relpath(target, ctx, entry_point_file)
-        properties["build.deps"] = ",".join(_crate_build_deps_info(crate_info))
+        if len(_crate_build_deps_info(crate_info)) > 0:
+            fail("Build deps support unimplemented")
+            #properties["build.deps"] = ",".join(_crate_build_deps_info(crate_info))
```

Comment:
We're still trying to parse build.deps in the ManifestSyncer. If we truly don't support them, that handling should be removed too.